guides/common/modules/proc_applying-errata-to-multiple-hosts-by-using-cli.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="applying-errata-to-multiple-hosts-by-using-cli"]
+= Applying errata to multiple hosts by using CLI
+
+You can use Hammer CLI to review and apply errata to multiple hosts.
+
+.Procedure
+. List all installable errata:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer erratum list \
+--errata-restrict-installable true \
+--organization "_My_Organization_"
+----
+. Apply one of the errata to multiple hosts.
++
+Using `Remote Execution`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ hammer job-invocation create \
+--feature katello_errata_install \
+--inputs errata=_My_Erratum_ID_ \
+--search-query "applicable_errata = _My_Erratum_ID_"
+----
++
+Using a Bash script that applies an erratum to each host for which this erratum is available:
++
+[source,terminal,subs="+quotes"]
+----
+for HOST in `hammer --csv --csv-separator "|" host list --search "applicable_errata = _My_Erratum_ID_" --organization "_My_Organization_" | tail -n+2 | awk -F "|" '{ print $2 }'` ;
+do
+  echo "== Applying to $HOST ==" ; hammer host errata apply --host $HOST --errata-ids _My_Erratum_ID_1_,_My_Erratum_ID_2_ ;
+done
+----
```

Comment:
This section is out of date. `hammer host errata apply` only worked with katello-agent. Nowadays if you run it you see: 
```
Not supported. Use the remote execution equivalent `hammer job-invocation create --feature katello_errata_install`.
```
This helps solidify that the CLI modules could be combined with a better explanation about how the search query works. Applying errata to multiple hosts is the same as applying errata to a single host, the only thing that differs is the search query.

Perhaps then there should just be a single "Applying errata to hosts by using CLI" module that mentions using the search query and links to docs that explains its use in more detail?